container_files/guac/guac.yaml

```diff
@@ -1,6 +1,9 @@
 # nats
 nats-addr: nats://nats:4222
 
+# blob store setup. Setup with blob store of choice via https://gocloud.dev/howto/blob/
+blob-addr: file:///path/to/dir
+
```

Comment:
We should probably have a shared volume/directory between the host and ingestor container and have this point to that. Can do this as a followup.

Reply:
Created an issue to capture this and will do in another PR: https://github.com/guacsec/guac/issues/1650